pkg/controllers/controlplanemachineset/consts.go

```diff
@@ -127,6 +127,11 @@ const (
 	// This will typically occur when an old replica has not yet been removed.
 	reasonExcessReplicas = "ExcessReplicas"
 
+	// reasonExcessUpdatedReplicas denotes that the ControlPlaneMachineSet has a more than expected number
+	// of ready and up-to-date replicas.
+	// This will typically occur when extra replica(s) for an index has been created.
```

Comment:
Nit, `an extra replica has`, vs, `extra replicas have`, given we are missing `an` I think `have`
```suggestion
	// This will typically occur when extra replica(s) for an index have been created.
```